hudi-common/src/test/java/org/apache/hudi/common/table/view/TestPriorityBasedFileSystemView.java

```diff
@@ -75,7 +75,8 @@ public class TestPriorityBasedFileSystemView {
 
   @BeforeEach
   public void setUp() {
-    fsView = new PriorityBasedFileSystemView(primary, secondary);
+    fsView = new PriorityBasedFileSystemView(primary, () -> secondary);
+    fsView.getSecondaryView();
```

Comment:
why init 2nd view here? then it's not testing lazy init, is it?

Reply:
there are many following cases use secondary directly, if not init here, will cause issues.

Reply:
done

Reply:
Hi @xushiyan , I have added lazy test for this PR, please review again